tests/unit/rest/worksheets_test.py

```diff
@@ -28,3 +30,10 @@ def test_create(self):
                 }
             ],
         )
+        # Verify the date_created and date_last_modified field has been set to the current time for a new created worksheet
+        worksheet_info = self.app.get('/rest/interpret/worksheet/' + worksheet_id).json
```

Comment:
Can you use freezegun to manually set the time (see this for an example: https://github.com/codalab/codalab-worksheets/blob/d403edc937ff598d930bbc13b53cb3922af9c4c2/tests/unit/rest/bundles_test.py#L5)

This way, you wouldn't have to use assertAlmostEqual -- you can just use assertEqual (and then the test would be less flaky since it doesn't have to take one second).

Reply:
Actually, this isn't super important -- we could maybe just do this in a later PR.